index.bs

```diff
@@ -5936,7 +5999,7 @@ The attributes above are structured within this certificate as such:
   04 12                                  -- OCTET STRING
     04  10                               -- OCTET STRING
       CD 8C 39 5C 26 ED EE DE            -- AAGUID cd8c395c-26ed-eede-653b-00797d03ca3c
-      65 3B 00 79 7D 03 CA 3C 
+      65 3B 00 79 7D 03 CA 3C
```

Comment:
unintentional change?

Reply:
I've replaced the space for sake of minimizing PR content.